### PR TITLE
CIV-12364 - Notify LiP Claimant that Translated Docs are uploaded.

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-12364

### Change description ###
Added New notification for LiP Claimant .

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by kenneth-hmcts — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedClaimantNotificationHandler.java [hunk 1]
```diff
@@ -61,18 +68,34 @@ CLAIM_LEGAL_ORG_NAME_SPEC, getApplicantLegalOrganizationName(caseData)
...
+        if (caseData.isLipvLipOneVOne() && featureToggleService.isLipVLipEnabled()) {
+            if (caseData.isBilingual()) {
+                return notificationsProperties.getNotifyClaimantLiPTranslatedDocumentUploadedWhenClaimIssuedInBilingual();
+            }
+            return notificationsProperties.getNotifyClaimantLiPTranslatedDocumentUploadedWhenClaimIssuedInEnglish();
+        }
+        return notificationsProperties.getNotifyClaimantTranslatedDocumentUploaded();
+    }
+
+    private String addEmail(CaseData caseData) {
```
Comment (manish14836) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedClaimantNotificationHandler.java:93]: can you please rename this method. This is not adding email anywhere.

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -345,6 +345,10 @@ notifications:
 
   notifyRespondentLipForClaimantRepresentedTemplate: "54dcfaca-41df-4588-bf3d-bf63f8d34570"
   notifyClaimantLipForNoLongerAccessTemplate: "8bf1b6c6-2ee7-442d-8b72-550d2aca9bdf"
+
```
Comment (manish14836) [inline src/main/resources/application.yaml:348]: Remove this line

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedClaimantNotificationHandlerTest.java [hunk 3]
```diff
@@ -49,16 +48,21 @@ public class TranslatedDocumentUploadedClaimantNotificationHandlerTest extends B
     private OrganisationService organisationService;
     @MockBean
     private FeatureToggleService featureToggleService;
-    private final String emailTemplate = "template-id";
-    private final String claimantEmail = "applicantsolicitor@example.com";
+    private final String emailTemplate1 = "template-id";
+    private final String emailTemplate2 = "template-id-Bilingual";
```
Comment (manish14836) [inline src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedClaimantNotificationHandlerTest.java:52]: these variables can be static

**Review by paul-pearson — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedClaimantNotificationHandler.java (hunk 1 above)
Comment (neeta-hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedClaimantNotificationHandler.java]: done

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedClaimantNotificationHandlerTest.java (hunk 3 above)
Comment (neeta-hmcts) [reply src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedClaimantNotificationHandlerTest.java]: done

File: src/main/resources/application.yaml (hunk 2 above)
Comment (neeta-hmcts) [reply src/main/resources/application.yaml]: done

**Review by kenneth-hmcts — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by manish14836 — APPROVED** [review]

**Pull request merged**